pages/deployments/deploying_to_kubernetes.md.erb

```diff
@@ -0,0 +1,192 @@
+# Deploying to Kubernetes
+
+This tutorial teaches deploying to Kubernetes. You'll need to prepare your
+agents with the required packages, cluster access, and Docker registry access
+before hand.
+
+<%= estimated_time "30 minutes" %>
+
+<%= toc %>
+
+## Overview
+
+<%= image "final_test_pipeline.png" %>
+
+You'll automate deploys from `master` to Kubernetes by the end of the tutorial.
+
+This tutorial uses two pipelines. The first pipeline builds, tests, and pushes
+a Docker image to a registry. The second pipeline-our deploy pipeline-is
+provided a `DOCKER_IMAGE` environmentment for deployment to Kubernetes. You'll
+see how to trigger manual deployments, then link the pipelines for automated
+deployments.
+
+This tutorial assumes:
```

Comment:
I think 'pre-requisites' rather than 'assumptions' here, we've used pre-reqs in other places.